doc/classes/AnimationTreePlayer.xml

```diff
@@ -219,6 +232,7 @@
 			</argument>
 			<description>
 				Sets mix amount of a Mix node given its name and value.
+				A Mix node adds input b to input a by a the amount given by ratio.
```

Comment:
Please re-review this sentence. It's very confusing.

@cbscribe It'd be better to edit with ~strikethrough~ and some explanations instead of deleting the comment.